syntax = "proto3";

package cakelab.arflow_grpc.v1;

import "cakelab/arflow_grpc/v1/device.proto";
import "cakelab/arflow_grpc/v1/session.proto";
import "cakelab/arflow_grpc/v1/synchronized_ar_frame.proto";

option csharp_namespace = "CakeLab.ARFlow.Grpc.V1";

message SaveSynchronizedARFrameRequest {
  SessionUuid session_id = 1;
  Device device = 2;
  SynchronizedARFrame frame = 3;
}
